### PR TITLE
1.14/1871 new registrationpagetest tests

### PR DESCRIPTION
**Pull request by neonleann**
https://trello.com/c/DrhVj0Mn/1871-new-registrationpagetest-tests

### CODE REVIEW

**Review by lauragurney — APPROVED** [review]
Now that looks smashing. Great work. 🐻‍❄️

File: app/Http/Controllers/Service/Admin/SponsorsController.php [hunk 1]
```diff
@@ -245,7 +245,7 @@ public static function socialPrescribingOverrides()
                 "name" => "DeductFromCarer",
                 "value" => -7,
                 "purpose" => "credits",
-                "entity" => "App\Child",
+                "entity" => "App\Family",
```
Comment (charlesstrange2) [inline app/Http/Controllers/Service/Admin/SponsorsController.php:248]: aha! good spot

Comment (charlesstrange2) [reply app/Http/Controllers/Service/Admin/SponsorsController.php:248]: this entire array feels like it should be a data item in `config/arc`
similarly the rules in the evaluatorfactory

we should make a card to extract them

**Review by charlesstrange2 — APPROVED** [review]
🏁 okedoke 🚗

**Pull request merged**